core/templates/pages/exploration-editor-page/history-tab/services/compare-versions.service.spec.ts

```diff
@@ -45,20 +45,40 @@ import { WrittenTranslationsObjectFactory } from
 import { UpgradedServices } from 'services/UpgradedServices';
 // ^^^ This block is to be removed.
 
+import { HttpClientTestingModule, HttpTestingController } from
+  '@angular/common/http/testing';
+import { TestBed, fakeAsync, flushMicrotasks } from '@angular/core/testing';
+
+import { ReadOnlyExplorationBackendApiService } from
+  'domain/exploration/read-only-exploration-backend-api.service';
 import { TranslatorProviderForTests } from 'tests/test.extras';
 
 require(
   'pages/exploration-editor-page/history-tab/services/' +
   'compare-versions.service.ts');
 
 describe('Compare versions service', function() {
+  let readOnlyExplorationBackendApiService = null;
+  let httpTestingController = null;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [HttpClientTestingModule]
+    });
+    readOnlyExplorationBackendApiService = TestBed.get(
+      ReadOnlyExplorationBackendApiService);
+    httpTestingController = TestBed.get(HttpTestingController);
+  });
   beforeEach(angular.mock.module('oppia'));
   beforeEach(angular.mock.module('oppia', function($provide) {
     var ugs = new UpgradedServices();
     for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
       $provide.value(key, value);
     }
   }));
+  afterEach(() => {
+    httpTestingController.verify();
+  });
 
   describe('compare versions service', function() {
     var cvs = null;
```

Comment:
Ditto regarding `var` -> `let`.

Reply:
Will be done when we migrate this particular spec file.